build/DependencyVersions.props

```diff
@@ -12,7 +12,7 @@
     <!-- We'll usually want to keep these versions in sync, but we may want to diverge in some
          cases, so use separate properties but derive one from the other unless we want to
          explicitly use different versions. -->
-    <CLI_NETSDK_Version>2.0.1-servicing-20170926-1</CLI_NETSDK_Version>
+    <CLI_NETSDK_Version>2.0.2-vspre-20170921-5</CLI_NETSDK_Version>
```

Comment:
I think this is resolved wrong.

Reply:
Fixed, also sent PR to do same merge and bump SDK version to 2.0.3 to avoid confusion